src/expanding.js

```diff
@@ -0,0 +1,72 @@
+import { wrap, inputEvent, dispatch } from './helpers'
+import Textarea from './textarea'
+import TextareaClone from './textarea-clone'
+
+function Expanding (textarea) {
+  var _this = this
+  this.element = createElement()
+  this.textarea = new Textarea(textarea)
+  this.textareaClone = new TextareaClone()
+  this.textarea.oldStyleAttribute = textarea.getAttribute('style')
+  resetStyles.call(this)
+  setStyles.call(this)
+
+  wrap(textarea, this.element)
+  this.element.appendChild(this.textareaClone.element)
+
+  function inputHandler () {
+    _this.update.apply(_this, arguments)
```

Comment:
Could we do `var inputHandler = this.update.bind(this)` and then drop the _this variable?